src/bin/readstat.c

```diff
@@ -420,7 +445,13 @@ static int dump_metadata(readstat_metadata_t *metadata, void *ctx) {
     }
     if (timestamp) {
         char buffer[128];
+#if !defined _MSC_VER
         strftime(buffer, sizeof(buffer), "%d %b %Y %H:%M", localtime(&timestamp));
+#else
+        struct tm ltm;
+        localtime_s(&ltm, &timestamp);
+        strftime(buffer, sizeof(buffer), "%d %b %Y %H:%M", &ltm);
+#endif
```

Comment:
For a more complicated platform difference like this, a separate function is probably warranted, e.g.

```
size_t readstat_strftime(char *s, size_t maxsize, const char *format, time_t timsetamp) {
#if !defined _MSC_VER
...
#else
...
#endif
}
```

That way we keep _MSC_VER out of the main application functions.